spec/rubocop/cop/lint/useless_trailing_comma_spec.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::UselessTrailingComma do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  context 'curly brace block format' do
+    it 'registers an offense when a trailing comma is not needed' do
+      expect_offense(<<~RUBY)
+        test { |a, b,| a + b }
+               ^^^^^^^ Useless trailing comma present in block arguments.
+      RUBY
```

Comment:
You can use `expect_correction` to keep the offense and correction testing as close together as possible. This will also avoid repeating the source of the code in multiple places.

The test `auto-corrects useless commas` can be removed entirely in favor of adding the following to this test.

```ruby
expect_correction(<<~RUBY)
  test { |a, b| a + b }
RUBY
```